oeps/processes/oep-0001.rst

```diff
@@ -415,6 +432,9 @@ OEP.
 * Additional diagrams or clarifying material (as long as the `Architecture
   Group`_ agrees that the substance of the OEP isn't changed).
 
+For the latter two points, please follow `Step 4. Announcing Changes`_ to inform
```

Comment:
Yeah I meant, of the above four bullets, the latter two (so, points 3 & 4). So not if _any_ changes are made, but like, any substantive changes. How about,

```suggestion
When making changes larger than simple formatting or spelling corrections, please follow `Step 4. Announcing Changes`_ to inform
```